packages/office-ui-fabric-react/src/components/Facepile/Facepile.tsx

```diff
@@ -142,7 +142,7 @@ export class Facepile extends React.Component<IFacepileProps, {}> {
   private _getAddNewElement(): JSX.Element {
     let { personaSize } = this.props;
     return <button { ...getNativeProps(this.props.addButtonProps, buttonProperties) }
-      className={ css('ms-Persona', PERSONA_SIZE[personaSize], 'ms-Facepile-addButton', 'ms-Facepile-itemButton') }>
+      className={ css('ms-Persona', PERSONA_SIZE[personaSize], 'ms-Facepile-addButton', 'ms-Facepile-itemButton', styles.addButton, styles.itemButton) }>
```

Comment:
I think this is a bug.. Facepile is creating a button with ms-Persona class, so it's relying on some external css here, and PERSONA_SIZE[personaSize] maps to some css class that won't exist once Persona is converted...